src/index.html

```diff
@@ -1,13 +1,403 @@
+          </a>
+
+          <a href="#sunglasses" class="products__category">
+            <div class="products__photo-box">
+              <div class="products__photo-container products__photo-container--1-2">
+                <img
+                  src="./images/photo5.png"
+                  alt="Audio sunglasses photo #1"
+                  class="products__product-photo"
+                >
+              </div>
+
+              <div class="products__photo-container products__photo-container--2-4">
+                <img
+                  src="./images/photo6.png"
+                  alt="Audio sunglasses photo #2"
+                  class="products__product-photo products__product-photo--position--right"
+                >
+              </div>
+            </div>
+
+            <h3 class="products__category-name">
+              Audio sunglasses
+            </h3>
+          </a>
+        </div>
+
+        <div class="products__button-box">
+          <a href="#view-all" class="products__button-view-all products__button-view-all--2-3 button">
+            View all
+          </a>
+        </div>
+      </section>
+
+      <section
+        class="corp__section corp__section--display--grid corp__section--margin--bottom buy-direct"
+        id="buy-direct"
+      >
+        <h2 class="buy-direct__title">
+          Why buy direct
+          <br>
+          from Bose
+        </h2>
+
+        <div class="buy-direct__about">
+          <div class="buy-direct__benefits">
+            <p class="buy-direct__strong">
+              Free 2-day shipping and returns
+            </p>
+
+            <p class="buy-direct__strong">
+              90-day risk-free trial
+            </p>
+
+            <p class="buy-direct__strong">
+              World class customer service
+            </p>
+
+            <p class="buy-direct__strong">
+              My Bose account management
+            </p>
+          </div>
+
+          <p class="buy-direct__description">
+            A great product is more than what’s in the box. It’s a promise of premium performance, world-class support, and
+            everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with confidence on
+            <a
+              href="https://www.bose.com/en_us/index.html"
+              class="buy-direct__link"
+              target="_blank"
+            >Bose.com</a>.
+          </p>
+        </div>
+      </section>
+
+      <section class="corp__section contacts" id="contacts">
+        <h2 class="contacts__title">
+          Contact us
+        </h2>
+
+        <div class="contacts__box">
+          <form
+            action="#"
+            method="post"
+            class="contacts__form form"
+          >
+            <label>
+              <input
+                type="text"
+                name="name"
+                placeholder="Name"
+                class="form__input"
+                required
+              >
+            </label>
```

Comment:
Is it necessary to wrap inputs by labels?

Reply:
![image](https://user-images.githubusercontent.com/105124305/188130944-69309d98-29d0-4fb1-ba6e-8a205135e5ac.png)

Input with id "null" has no associated label  input-req-label